Unix/installbuilder/GNUmakefile

```diff
@@ -159,6 +159,13 @@ else #($(PF),Darwin)
 	# copy normal libmi.so for other platforms
 	cp $(OUTPUTDIR)/lib/libmi.so $(PACKAGE_DIR)
   endif #ifeq ($(PF),Linux)
+ 
+  # copy some providers for Linux/Unix platforms
+  mkdir -p $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libalert.so $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libfilesys.so $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libPresidentProvider.so $(PACKAGE_DIR)/test
+  cp $(OUTPUTDIR)/lib/libTestClass_AllDMTFArrays.so $(PACKAGE_DIR)/test
```

Comment:
At least these are in a subdirectory, that's good. But yuck, that means that every single build, on every single platform, is going to need these. And there are a lot of test providers, this list can grow longer.

Do you really need these providers for each and every platform for each and every build? Remember, you can build these but not stage them. That way, you can just build a specific platform if needed, and it doesn't end up on the build share. It's cleanliness of the build share that is my biggest concern here.

If you REALLY need these for every single platform on every single build, then either we put these in a `test` directory, as you did here, or we put them in some other artifact directory (which Jenkins does support), and copy them to some other location.

Please give guidance of how often you realistically need these, and we'll go from there.